bundle/manifests/datadoghq.com_datadogagents.yaml

```diff
@@ -2489,6 +2489,10 @@ spec:
                           description: 'Enable informer and controller of the watermark pod autoscaler NOTE: The WatermarkPodAutoscaler controller needs to be installed see https://github.com/DataDog/watermarkpodautoscaler for more details.'
                           type: boolean
                       type: object
+                    kubeStateMetricsCoreEnabled:
+                      description: Enable this to start the Kubernetes State Metrics
+                        Core check. TODO add reference to the documentation.
+                      type: boolean
```

Comment:
I think this part should be present anymore,  since `enable` is now a field in the `kubeStateMetricsCore` section

Reply:
ah, odd that it still there

Reply:
updated the bundle... Seems like it was missing a few things from previous changes too